tests/test_rechunk.py

```diff
@@ -131,6 +242,86 @@ def test_rechunk_dataset(
     assert ds.attrs == dst.attrs
 
 
+@pytest.mark.parametrize("shape", [(100, 50)])
+@pytest.mark.parametrize("source_chunks", [(10, 50), (100, 5)])
+@pytest.mark.parametrize(
+    "target_chunks",
+    [
+        {"x": 20},  # ? Should this rechunk y? Probably not...
+        {"x": 20, "y": 1e5},
+        {"x": 20, "y": -1},
+    ],
+)
+@pytest.mark.parametrize("max_mem", ["10MB"])
+@pytest.mark.parametrize("target_store", ["target.zarr", "mapper.target.zarr"])
+@pytest.mark.parametrize("temp_store", ["temp.zarr", "mapper.temp.zarr"])
+def test_rechunk_dataset_dimchunks(
+    tmp_path, shape, source_chunks, target_chunks, max_mem, target_store, temp_store,
+):
+    if target_store.startswith("mapper"):
+        fsspec = pytest.importorskip("fsspec")
```

Comment:
Is fsspec in our test env? I'm confused why we are mixing in fsspec mapper stuff with this new feature about chunking.